apps/nextjs-website/src/app/page.tsx

```diff
@@ -35,6 +38,12 @@ const Home = async () => {
           svgPath: product.svgPath,
         }))}
       />
+      <WebinarsSection
+        title={homepage.webinarsSection.title}
+        description={homepage.webinarsSection.description}
+        link={homepage.webinarsSection.link}
```

Comment:
I think is better to remove link for now that webinars list page does not esist yet
```suggestion
```